pkg/agent/route/route_linux.go

```diff
@@ -512,6 +546,19 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet,
 				"-j", iptables.NoTrackTarget,
 			}...)
 		}
+
+		if c.multicastEnabled && c.networkConfig.TrafficEncapMode.SupportsEncap() {
+			// Drop the multicast packets forwarded from other Nodes in the cluster. This is because
+			// the packet sent out from the sender Pod is already received via tunnel port with encap mode,
+			// and the one forwarded via the underlay network is to send to external receivers
+			writeLine(iptablesData, []string{
+				"-A", antreaPreRoutingChain,
+				"-m", "comment", "--comment", `"Antrea: drop Pod multicast traffic forwarded via underlay network"`,
+				"-m", "set", "--match-set", clusterNodeIPSet, "src",
+				"-d", types.McastCIDR.String(),
+				"-j", iptables.DROPTarget,
+			}...)
```

Comment:
I think it's more typical to drop packet in filter table.

I also wonder if we should handle this in FORWARD chain of filter table, then the node itself can receive traffic from other Nodes, and local won't receive duplicate traffic.

Reply:
could you explain what the message means? and whether it's triggered every time a multicast packet is received or just the first one? and is my above assumption correct? "the node itself can receive traffic from other Nodes, and local Pods won't receive duplicate traffic."

Reply:
IGMP_NOCACHE message is triggered per packet if no multicast route entry matching the src and dst is found.